ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -242,6 +242,13 @@ IgvPanel.propTypes = {
   locus: PropTypes.string,
 }
 
+const TISSUE_REFERENCE_KEY = {
+  WB: 'Blood',
+  F: 'Fibs',
+  M: 'Muscle',
+  L: 'Lymph',
```

Comment:
Is there a reason not to just use `TISSUE_DISPLAY` here?

Reply:
The fields and values of the `TISSUE_DISPLAY` are:
```
  WB: 'Whole Blood',
  F: 'Fibroblast',
  M: 'Muscle',
  L: 'Lymphocyte',
```
Most of them don't match the names of the reference tracks, such as `GTEx Blood`, `GTEx Fibs`, and `GTEx Lymph`.